temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using print statements in committed code. If you need to output information, consider using the `logging` module for better control over log levels and output destinations.